packages/browser/src/eventbuilder.ts

```diff
@@ -87,6 +87,8 @@ export function eventFromUnknownInput(
 
     event = eventFromString(message, syntheticException, options);
     addExceptionTypeValue(event, message);
+    event.tags = domException.code ? { ...event.tags, 'DOMException.code': String(domException.code) } : event.tags;
```

Comment:
```suggestion
    if ('code' in domException) {
      event.tags = { ...event.tags, 'DOMException.code': `${domException.code}` }
    }
```

Reply:
I left the "old" tags, just in case we ever decided to for example mark events as being produced by `eventFromString` or something. Keep in mind that its safe to destructure undefined values, see:
https://github.com/tc39/proposal-object-rest-spread/blob/master/Spread.md
> Null/Undefined Are Ignored

Reply:
Wait, I'm confused. I did, too...